docs/reference/monitoring-with-elastic-agent.md

```diff
@@ -47,10 +47,14 @@ monitoring.cluster_uuid: PRODUCTION_ES_CLUSTER_UUID
 ::::
 
 
-::::{dropdown} Create a monitoring user (standalone agent only)
+::::{dropdown} Grant agent access to Elasticsearch (standalone agent only)
 :name: create-user-ea
 
-Create a user on the production cluster that has the `remote_monitoring_collector` [built-in role](elasticsearch://reference/elasticsearch/roles.md).
+The following minimal permissions are required to send Logstash monitoring data to Elasticsearch:
+
+`monitor` cluster privilege
+`auto_configure` and `create_doc` index privileges on `logs-*` and `metrics-*` indices.
+See [Grant standalone Elastic Agents access to Elasticsearch](docs-content://reference/fleet/grant-access-to-elasticsearch.md)
```

Comment:
```suggestion
- `monitor` cluster privilege
- `auto_configure` and `create_doc` index privileges on `logs-*` and `metrics-*` indices.

Check out [Grant standalone Elastic Agents access to Elasticsearch](docs-content://reference/fleet/grant-access-to-elasticsearch.md) for details. 
```

Reply:
I don't think that this is rendering as you intended: 

<img width="739" alt="Screenshot 2025-03-31 at 7 46 07 PM" src="https://github.com/user-attachments/assets/06a0273d-484a-4d0e-8c35-76a04ffb9957" />